.github/workflows/notebooks.yml

```
@@ -7,10 +7,29 @@ on:
     branches: [ main ]
 
 jobs:
-  build:
-
+  build_damask:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2.2.0
+      with:
+        python-version: "3.11"
+        mamba-version: "*"
+        miniforge-variant: Mambaforge
+        channels: conda-forge
+        channel-priority: strict
+        auto-update-conda: true  
+        environment-file: .ci_support/environment.yml
+    - name: Setup
+      shell: bash -l {0}
+      run: |
+        pip install --no-deps .
+        mamba env update --name test --file .ci_support/environment-notebooks-damask.yml
```

I think both of these can be replaces with the [`cached-mamba`](https://github.com/pyiron/actions/blob/main/cached-mamba/action.yml) centralized action. You might need to debug this syntax, but off the top of my head I think it should just be

```suggestion
  - uses: pyiron/actions/cached-mamba@main
    with:
      python-version: "3.11"
      env-prefix: "ubuntu-latest"
      env-label: "ubuntu-latest-py3_11" 
      env-files: .ci_support/environment.yml .ci_support/environment-notebooks-damask.yml
```

(or `.ci_support/environment.yml .ci_support/environment-notebooks-shared.yml .ci_support/environment-notebooks-damask.yml` per my other suggestion)